os/fs/smartfs/Kconfig

```diff
@@ -3,6 +3,14 @@
 # see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
 #
 
+config CONFIG_SMART_FS_SECURE
+	bool "Secure File System"
+	default n
+	select CRYPTO
+	select CRYPTO_SW_AES
```

Comment:
Don't you need to make a dependancy with ```FS_SMARTFS```? Does it work independently?

Reply:
Yes, dependency is also required with FS_SMARTFS.